.planning/codebase/CONVENTIONS.md

```
@@ -0,0 +1,349 @@
+    public DateTime? UpdatedAt { get; set; }
+    public DateTime? DeletedAt { get; set; }
+
+    public uint Version { get; set; }  // Concurrency token
+
+    public ICollection<UserOrganization> Organizations { get; set; } = new List<UserOrganization>();
+}
+```
+
+## Endpoint Design
+
+**Minimal API Pattern:**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)